build.sbt

```diff
@@ -52,7 +62,12 @@ lazy val itemImpl = (project in file("item-impl"))
     )
   )
   .settings(lagomForkedTestSettings: _*)
-  .dependsOn(tools % "test -> test", itemApi, biddingApi)
+  .dependsOn(
+    tools % "test -> test",
+    testkit % "test -> test",
```

Comment:
Using @TimMoore 's suggestion this dependency will be simplified to `testkit % "test"`.
Moving `ReadSideTestDriver` to `testkit` will also simplify ` tools % "test -> test"` into `tools`.

I'm liking this split into `tools` and `testkit`. 👍